converter/src/main/java/gov/cms/qpp/conversion/decode/ClinicalDocumentDecoder.java

```diff
@@ -30,6 +30,7 @@
 	public static final String ENTITY_TYPE = "entityType";
 	public static final String MIPS_PROGRAM_NAME = "mips";
 	public static final String CPCPLUS_PROGRAM_NAME = "cpcPlus";
+	public static final String PCF = "PCF";
```

Comment:
I thought talking with Michelle she was going to keep this all lowercase. Originally it was case insensitive so any caps was going to be accepted, but we talked her down to all lowercase. That said, I see the other variables not being appended with `_PROGRAM_NAME` are always in caps so might be some context I'm missing...

Reply:
To be safe, I changed the conversion to use lowercasing as Conversion Tool's validation only ensures the value is PCF regardless of case sensitivity.